bin/sdk-cli.js

```diff
@@ -2,22 +2,87 @@
 
 /** @format */
 
+/**
+ * External dependencies
+ */
 const chalk = require( 'chalk' );
 const path = require( 'path' );
 const spawnSync = require( 'child_process' ).spawnSync;
+const yargs = require( 'yargs' );
 
 const buildBlockScript = path.resolve( __dirname, 'create-scripts/block.js' );
-const [ /* node */, /* bin/sdk-cli.js */, task, ...args ] = process.argv;
+const extensionsDir = path.resolve( __dirname, '../client/gutenberg/extensions' );
+const prePackagedBlocks = [
+	'editor-notes',
+	'hello-dolly',
+];
 
-if ( ! args.length || task !== 'build-block' ) {
-	console.log( chalk.red( 'usage: npx calypso-gutenberg-sdk build-block <block>' ) );
-	process.exit(1);
-}
+const buildBlock = argv => {
+	let entryFile;
 
-spawnSync( 'node', [ buildBlockScript, ...args ], {
-	env: {
-		SKIP_FLAG_IMAGES: true,
-	},
-	shell: true,
-	stdio: 'inherit',
-} );
+	if ( argv.block ) {
+		switch ( argv.block ) {
+			case 'editor-notes':
+				entryFile = path.join( extensionsDir, 'editor-notes/index.js' );
+				break;
+			case 'hello-dolly':
+				entryFile = path.join( extensionsDir, 'hello-dolly/hello-block.js' );
+				break;
+			default:
+				console.log( chalk.red( `Unknown block "${ argv.block }" - list available blocks with "list-blocks"` ) );
+				process.exit( 1 );
```

Comment:
So these definitely shouldn't stay here for long but I added them because I didn't want to use the same `--editor-js` for both blocks that come with Calypso and blocks that developer wants to build from anywhere in their filesystem.

When: `--editor-js=client/gutenberg/extensions/hello-dolly/hello-block.js`

```js
path.resolve( __dirname, 'client/gutenberg/extensions/hello-dolly/hello-block.js' )
// → resolves to ~/jetpack/node_modules/wp-calypso/client/ ...
// We would assume all the blocks live always inside Calypso
```

```js
path.resolve( 'client/gutenberg/extensions/hello-dolly/hello-block.js' )
// → resolves to ~/jetpack/client/ ...
// We would assume all the blocks live always outside Calypso
```

We couldn't have both with the same `--editor-js` and hence I added `--block hello-world` which is a bit of magic that resolves things inside Calypso on its own. Details for different blocks should not be hardcoded inside the cli file but we don't have any mechanism yet thought out, so here they are.

See full Slack convo: p1532596071000242-slack-calypso-gutenberg